app/src/main/java/cm/aptoide/pt/install/InstallAnalytics.java

```diff
@@ -28,13 +29,16 @@
   public static final String APPLICATION_INSTALL = "Application Install";
   public static final String EDITORS_APPLICATION_INSTALL = "Editors_Choice_Application_Install";
   public static final String INSTALL_EVENT_NAME = "INSTALL";
+  private static final int MIGRATION_UNINSTALL_KEY = 8726;
```

Comment:
What is this? this will be used as installingVersion, what are the effects of setting this specific number?